components/Article.js/Article.js

```diff
@@ -14,19 +23,28 @@ export default function Article() {
 
   const stories = data.news.filter(
     (item) =>
-      item.type === 'story' && item.teaserImage?.imageVariants?.['16x9-256']
+      item.type === "story" && item.teaserImage?.imageVariants?.["16x9-256"]
   );
 
   return (
     <div>
-      {stories.map(({ title, teaserImage, shareURL }, sophoraId) => (
-    
-        <StyledContainer key={sophoraId}>
-          <StyledH1>{title}</StyledH1>
-          <StyledImage src={teaserImage.imageVariants?.['16x9-256']} />
-          <StyledLink href={shareURL} target="_blank" rel="noopener noreferrer">
-          zum Artikel
-        </StyledLink>
+      {stories.map(({ title, teaserImage, shareURL }) => (
+        <StyledContainer key={shareURL}>
+          <MainContainer>
+            <StyledH1>{title}</StyledH1>
+            <ContentContainer>
+            <StyledImage src={teaserImage.imageVariants?.["16x9-256"]} />
+            <ButtonContainer>
+            <FavoriteButton article={{ title, teaserImage, shareURL }} /> 
```

Comment:
why does your favorite-button need the title, teaserImage and shareURL? In my opinion the ``sophoralId``would be nice to store in LocalStorage and you can use it anywhere else to render the corresponding articles.